backend/oidc/urls.py

```diff
@@ -1,17 +1,44 @@
+from django.conf import settings
 from django.urls import include, path
 
+from oidc.mock_views import (
+    MockAuthenticationRequestView,
+    MockLogoutView,
+    MockUserInfoView,
+)
 from oidc.views import HelsinkiOIDCLogoutView, HelsinkiOIDCUserInfoView
 
-urlpatterns = [
-    path(
-        "logout/",
-        HelsinkiOIDCLogoutView.as_view(),
-        name="oidc_logout",
-    ),
-    path(
-        "userinfo/",
-        HelsinkiOIDCUserInfoView.as_view(),
-        name="oidc_userinfo",
-    ),
-    path("", include("mozilla_django_oidc.urls")),
-]
+urlpatterns = []
+
+if settings.MOCK_FLAG:
```

Comment:
could the flag name be more verbose, like AUTH_MOCK_FLAG?

Do we need to set default value (False) for the flag in settings.py?

Reply:
hmm on the second thought, it is fine to use the same MOCK_FLAG that the company api already uses